tests/test_forecast.py

```
@@ -8,39 +8,58 @@
 import datetime
 
 from forecast import forecast
+import forecast.redis as redis_module
```

Ruff

`forecast.redis` imported but unused

Remove unused import: `forecast.redis`

(F401)